altair/vegalite/v5/compiler.py

```diff
@@ -0,0 +1,26 @@
+from ..compiler import VegaLiteCompilerRegistry
+
+vegalite_compilers = VegaLiteCompilerRegistry()
```

Comment:
We could already add an entry point group so that it's easier to register new compilers.

```suggestion
ENTRY_POINT_GROUP: str = "altair.vegalite.v5.vegalite_compiler"
vegalite_compilers = VegaLiteCompilerRegistry(entry_point_group=ENTRY_POINT_GROUP)
```

Reply:
Done in c7e860774eea5468c612e486767a38a8a64acc77